python/common/thief_constants.py

```diff
@@ -3,7 +3,91 @@
 # full license information.
 
 
-class ServiceAckType(object):
-    TELEMETRY_SERVICE_ACK = "telemetry"
-    ADD_REPORTED_PROPERTY_SERVICE_ACK = "add_reported"
-    REMOVE_REPORTED_PROPERTY_SERVICE_ACK = "remove_reported"
+class Const(object):
+    JSON_CONTENT_TYPE = "application/json"
+    JSON_CONTENT_ENCODING = "utf-8"
+    JSON_TYPE_AND_ENCODING = {
+        "contentType": "application/json",
```

Comment:
Can this value reference JSON_CONTENT_TYPE?

Reply:
not cleanly.  There's a few different ways to hack it, but they would only add confusion.